src/index.tsx

```diff
@@ -0,0 +1,9 @@
+import Honeybadger from "@honeybadger-io/js"
```

Comment:
Exporting _core_ `Honeybadger` from `@honeybadger-op/js` to make react integration simpler.